asv_bench/benchmarks/stat_ops.py

```diff
@@ -20,6 +20,38 @@ def time_op(self, op, dtype, axis):
         self.df_func(axis=axis)
 
 
+class FrameMixedDtypesOps:
+    params = [ops, [0, 1, None]]
+    param_names = ["op", "axis"]
+
+    def setup(self, op, axis):
+        N = 1_000_000
+        df = pd.DataFrame(
+            {
+                "f": np.random.normal(0.0, 1.0, N),
+                "i": np.random.randint(0, N, N),
+                "ts": pd.date_range(start="1/1/2000", periods=N, freq="h"),
+            }
+        )
+        if op in ("sum", "skew", "kurt", "prod", "sem", "var") or (
```

Comment:
Could you put this `if` statement before `N` and `df` are created?

Reply:
moved.
ci on arm failed on `test_dti_constructor_with_non_nano_now_today`. seems like a flaky test tho